### DISCUSSION TITLE
[Question] Using surface gripper in orbit possible?

### DISCUSSION THREAD

**Discussion by ValM99** [isaac-sim/IsaacLab#1106] [Q&A]
Hi,

is there a way to use surface gripper in orbit? So far I only found the 2-finger-gripper but i would like to use the surface gripper for picking up objects. I searched in the documentation and in the code but there were no examples.

Preferably I would like to implement it in my InteractiveSceneCfg class. Can you tell me if the usage of surface gripper is already possible now or if it will only work in the future?

**Comment by hhansen-bdai** [comment]
Currently it is possible to use the surface gripper but it will CPU only, unable to utilize the GPU.
